tp0_install/install.sh

```diff
@@ -1,7 +1,6 @@
 #! /bin/sh
 
-### Install basic dependencies 
-sudo apt-get install -y cmake cmake-curses-gui cmake-qt-gui curl doxygen g++ ipython libassimp-dev assimp-utils libboost-dev omniidl-python omniorb omniorb-idl omniorb-nameserver libomniorb4-dev libomnithread3-dev python python-matplotlib python-numpy python-scipy python2.7 qgit libbz2-dev zlib1g-dev libssl-dev pax tar libeigen3-dev libtinyxml-dev liburdfdom-dev libboost-all-dev libpcre3-dev libopenscenegraph-dev libqt4-dev python-qt4-dev libncurses-dev python-pip cython python-h5py
+### Setup environment
```

Comment:
1. we should keep the list of dependencies, should we not. Even if not directly useful, it seems to be an important part of the documentation.
2. Are you sure that none of these packets are useful. For example, h5py is needed for running the install of tensor-flow, is it not?